CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# 1.0.0
```

Comment:
We need to re-generate the CHANGELOG with the release_ready script since there has been merged changed since the first creation of this CHANGELOG.

Reply:
@pietroid don't worry about this, I'll take care of it and plan to release today.